### ISSUE TITLE
I see login page

### ISSUE THREAD

**Issue by jlopezjuy** [swathisprasad/spring-boot-angular6-maven-project#7]
Hi, i have cloned, run mvn clean install, next run and for some reason i see login page, it´s ok?
i have tested on linux and windows.

**Comment by swathisprasad** [comment]
Yes, that is due to spring-boot-starter-security dependency. To login use username: 'user' and copy paste password from console log generated when the application starts.

**Comment by jlopezjuy** [comment]
Do you know how to disable? i have remove dependency but login still working.
